sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -88,6 +89,8 @@ public void define(Context context) {
       addPhpStanExtensions(context);
       addPsalmExtensions(context);
     }
+
+    context.addExtension(DefaultAnalysisWarningsWrapper.class);
```

Comment:
This step is necessary to provide the wrapper also for the PhpSensor. Due to missing support of AnalysisWarnings in SonarLint we have to add a lot of alternative constructors to the sensor. I would prefer to have a separated PHPUnit sensor that is only executed in a non-SL context. Then the DefaultAnalysisWarningsWrapper extension could only be added in a non-SL context.